aten/src/ATen/native/native_functions.yaml

```diff
@@ -1961,6 +1961,10 @@
     CUDA: batch_norm_cuda
     MkldnnCPU: mkldnn_batch_norm
 
+- func: native_batch_norm.out(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float momentum, float eps, *, Tensor(a!) out, Tensor(b!) save_mean, Tensor(c!) save_invstd) -> (Tensor(a!), Tensor(b!), Tensor(c!))
+  dispatch:
+    CUDA: batch_norm_cuda_out
```

Comment:
By the way, since this is an internal function I'm not going to block the PR over this, but ideally there'd also be a CPU and MkldnnCPU dispatch entry too.